client/src/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,153 @@
+import { useState } from 'react';
+import { AppBar, Toolbar, Avatar, Button, Box } from '@material-ui/core';
+import Menu from '@material-ui/core/Menu';
+import { useAuth } from '../../context/useAuthContext';
+import logo from '../../Images/logo.png';
+import profile from '../../Images/profile.png';
+import useStyles from './useStyles';
+import { useHistory } from 'react-router-dom';
+import ArrowDropDownIcon from '@material-ui/icons/ArrowDropDown';
+
+const Navbar = (): JSX.Element => {
+  const { logout } = useAuth();
+  const classes = useStyles();
+  const history = useHistory();
```

Comment:
You can de-structure the location object in the same line

Reply:
Fixed this!!!